splice_spark2/src/main/scala/com/splicemachine/nsds/kafka/KafkaAdmin.scala

```diff
@@ -17,28 +17,37 @@ package com.splicemachine.nsds.kafka
 
 import java.util.Properties
 import java.util.concurrent.TimeUnit
-import org.apache.kafka.clients.admin.AdminClient
-import org.apache.kafka.clients.admin.AdminClientConfig
+
+import org.apache.kafka.clients.admin.{AdminClient, AdminClientConfig, NewTopic}
 import org.apache.kafka.common.KafkaFuture
+
 import scala.collection.JavaConverters._
 
-class KafkaAdmin(kafkaServers: String) {
+@SerialVersionUID(20200722241L)
+class KafkaAdmin(kafkaServers: String) extends Serializable {
   val props = new Properties()
   props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, kafkaServers)
   val admin = AdminClient.create( props )
   
+  def createTopics(topicNames: collection.mutable.Set[String], numParitions: Int, repFactor: Short): Unit = {
+    admin.createTopics(
+      topicNames.map(new NewTopic(_,numParitions,repFactor)).asJava
+    ).all.get
+//    Thread.sleep(1000)
```

Comment:
remove

Reply:
Made change in a temporary branch for review while regression tests are running, and will push to this branch after regression is finished.

https://github.com/splicemachine/spliceengine/blob/0263ed86b9c614882b6cde88a47c3caf87ca7027/splice_spark2/src/main/scala/com/splicemachine/nsds/kafka/KafkaAdmin.scala#L32-L36

Reply:
Pushed code to this branch.